src/webapp/github.py

```diff
@@ -16,12 +18,70 @@ def mk_github_authstr(user, passwd):
     raw = '%s:%s' % (user, passwd)
     return base64.encodebytes(raw.encode()).decode().replace('\n', '')
 
+
 def api_path2url(api_path, **kw):
     fmtstr = re.sub(r':([a-z]+)\b', r'{\1}', api_path)
     path = fmtstr.format(**kw)
     url = api_baseurl + path
     return url
 
+
+def http_to_dict(success, http_response, throw_error=True):
+    """http json response to dict"""
+
+    if success:
+        return json.load(http_response.fp)
+
+    if throw_error:
+
+        if http_response == "Not Found":
+            raise GithubNotFoundException(f"Request not successful: {str(http_response)}")
```

Comment:
I believe `str(x)` is implicit for `f"{x}"` (as well as `"{}".format(x)` and `"%s" % x`)

```suggestion
            raise GithubNotFoundException(f"Request not successful: {http_response}")
```